docs/prepare_prior_versions.py

```diff
@@ -534,20 +534,15 @@ def prepare_prior_version(version: Version) -> None:
     if version >= Version(0, 16):
         prepend_version_info_to_name_for_md_images(version)
 
-    # TODO: delete this after we cut 0.18.9
```

Comment:
We've already done this processing, so this is unneeded. (Really most of this file is, but we can address that separately)